### ISSUE TITLE
[Feature request] Reset best loss when restoring from checkpoint in order to keep best model of fine-tuning

### ISSUE THREAD

**Issue by alpoktem** [coqui-ai/Trainer#65]
**🚀 Feature Description**

I'm fine-tuning a Coqui TTS model on a different dataset and language. Since it's never reaching the best loss of the pre-trained model it never saves the best model (of the fine-tuning stage). I am only able to see the final checkpoints saved but the best model gets lost in history since I keep only the final X checkpoints. 

**Solution**

If I could reset the best loss saved in the model, I believe it would be possible to save the best model starting from the fine-tuning training.

**Comment by erogol** [comment]
I think @Edresson implemented that with the ```--restore_model``` flag.
